.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -1,52 +1,48 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows a user to place an order and an admin to list it', () => {
-  cy.visit('/desktops/mac');
+it('allows a user to add an item and register', () => {
+  cy.visit('/en-gb/product/macbook');
   cy.contains('Add to Cart').click();
   cy.contains('You have added');
   cy.visit('/index.php?route=checkout/checkout');
-  cy.contains('Guest Checkout').click();
-  cy.get('#button-account').click();
-  cy.fixture('guests').then((guest) => {
-    cy.get('#input-payment-firstname').type(
-      `${guest.newGuest.firstName}.${random}`
+  cy.fixture('customers').then((customer) => {
+    cy.get('#input-firstname').type(
+      `${customer.newCustomer.firstName}.${random}`
     );
-    cy.get('#input-payment-lastname').type(
-      `${guest.newGuest.lastName}.${random}`
+    cy.get('#input-lastname').type(
+      `${customer.newCustomer.lastName}.${random}`
     );
-    cy.get('#input-payment-email').type(`${guest.newGuest.email}`);
-    cy.get('#input-payment-telephone').type(guest.newGuest.phone);
-    cy.get('#input-payment-address-1').type(`${guest.newGuest.address}`);
-    cy.get('#input-payment-city').type(guest.newGuest.city);
-    cy.get('#input-payment-postcode').type(guest.newGuest.postCode);
-    cy.get('#input-payment-zone').select(guest.newGuest.zone);
+    cy.get('#input-email').type(`${customer.newCustomer.email}`);
+    cy.get('#input-password').type(`${customer.newCustomer.password}`);
+    cy.get('#input-shipping-address-1').type(`${customer.newCustomer.address}`);
+    cy.get('#input-shipping-city').type(customer.newCustomer.city);
+    cy.get('#input-shipping-postcode').type(customer.newCustomer.postCode);
+    cy.get('#input-shipping-zone').select(customer.newCustomer.zone);
   });
-  cy.get('#button-guest').click();
-  cy.get('#button-shipping-method').click();
-  cy.get('[name="agree"]').check();
-  cy.get('#button-payment-method').click();
-  cy.get('#button-confirm').click();
-  cy.contains('order has been placed');
+  cy.get('#agree').click();
+  cy.get('#button-register').click();
+  cy.contains('Your account has been created');
   cy.login();
-  cy.contains('Sales').click();
-  cy.contains('Orders').click();
-  cy.fixture('guests').then((guest) => {
-    cy.contains(`${guest.newGuest.firstName}.${random}`);
+  cy.contains('Customers').click();
+  cy.contains(`ul li ul li a`, 'Customers').click();
+  cy.fixture('customers').then((customer) => {
+  cy.contains(`${customer.newCustomer.email}`);
   });
 });
 
 it('allows an admin to add a product to the catalog', () => {
   cy.login();
   cy.contains('Catalog').click();
   cy.contains('Product').click();
-  cy.get('[data-original-title="Add New"]').click();
+  cy.get('a.btn.btn-primary i.fa-solid.fa-plus').click();
```

Comment:
Same as above, we should use a single selector if possible. As a suggestion you could test:

```suggestion
  cy.get('[aria-label="Add New"]').click();
```

Reply:
This should be already changed